Resources/Prototypes/_Goobstation/GameRules/Dynamic/dynamic.yml

```
@@ -84,7 +80,7 @@
     - Dynamic
     - BasicStationEventScheduler
     - DynamicStationEventScheduler
-    #- MeteorSwarmScheduler Goobstation - nuh uh
+    #- MeteorSwarmScheduler # Goobstation - nuh uh
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)